### PR TITLE
Px density docs

### PR DESCRIPTION
**Pull request by nicolaskruchten**
Missing docs for two PX functions! This forum post gets a ton of clicks and we never documented the answer: https://community.plotly.com/t/plotly-express-heatmap/21857

### CODE REVIEW

File: doc/python/2D-Histogram.md [hunk 1]
```diff
@@ -33,6 +33,69 @@ jupyter:
...
+
+## Density Heatmaps with Plotly Express
+
+[Plotly Express](/python/plotly-express/) is the easy-to-use, high-level interface to Plotly, which [operates on "tidy" data](/python/px-arguments/) and produces [easy-to-style figures](/python/styling-plotly-express/). The Plotly Express function `density_heatmap()` can be used to produce density heatmaps.
```
Comment (emmanuelle) [inline doc/python/2D-Histogram.md:42]: let's not mention tidy data here since this will change soon... one less file to change

**Comment by emmanuelle** [comment]
Any chance to remove the gremlin page https://plotly.com/python/2d-density-plots/ at the same time?

File: doc/python/2d-histogram-contour.md [hunk 2]
```diff
@@ -33,6 +33,69 @@ jupyter:
     thumbnail: thumbnail/hist2dcontour.png
 ---
 
+## 2D Histogram Contours or Density Contours
+
+A 2D histogram contour plot, also known as a density contour plot, is a 2-dimensional generalization of a [histogram](/python/histograms/) which resembles a [contour plot](/python/contour-plots/) but is computed by grouping a set of points specifed by their `x` and `y` coordinates into bins, and applying an aggregation function such as `count` or `sum` (if `z` is provided) to compute the value to be used to compute contours. This kind of visualization (and the related [2D histogram, or density heatmap](/python/2d-histogram/)) is often used to manage over-plotting, or situations where showing large data sets as [scatter plots](/python/line-and-scatter/) would result in points overlapping each other and hiding patterns.
```
Comment (emmanuelle) [inline doc/python/2d-histogram-contour.md:38]:
```suggestion
A 2D histogram contour plot, also known as a density contour plot, is a 2-dimensional generalization of a [histogram](/python/histograms/) which resembles a [contour plot](/python/contour-plots/) but is computed by grouping a set of points specified by their `x` and `y` coordinates into bins, and applying an aggregation function such as `count` or `sum` (if `z` is provided) to compute the value to be used to compute contours. This kind of visualization (and the related [2D histogram, or density heatmap](/python/2d-histogram/)) is often used to manage over-plotting, or situations where showing large data sets as [scatter plots](/python/line-and-scatter/) would result in points overlapping each other and hiding patterns.
```

File: doc/python/2D-Histogram.md [hunk 3]
```diff
@@ -33,6 +33,69 @@ jupyter:
     thumbnail: thumbnail/histogram2d.jpg
 ---
 
+## 2D Histograms or Density Heatmaps
+
+A 2D histogram, also known as a density heatmap, is the 2-dimensional generalization of a [histogram](/python/histograms/) which resembles a [heatmap](/python/heatmaps/) but is computed by grouping a set of points specifed by their `x` and `y` coordinates into bins, and applying an aggregation function such as `count` or `sum` (if `z` is provided) to compute the color of the tile representing the bin. This kind of visualization (and the related [2D histogram contour, or density contour](https://plotly.com/python/2d-histogram-contour/)) is often used to manage over-plotting, or situations where showing large data sets as [scatter plots](/python/line-and-scatter/) would result in points overlapping each other and hiding patterns. For data sets of more than a few thousand points, a better approach than the ones listed here would be to [use Plotly with Datashader](/python/datashader/) to precompute the aggregations before displaying the data with Plotly.
```
Comment (emmanuelle) [inline doc/python/2D-Histogram.md:38]:
```suggestion
A 2D histogram, also known as a density heatmap, is the 2-dimensional generalization of a [histogram](/python/histograms/) which resembles a [heatmap](/python/heatmaps/) but is computed by grouping a set of points specified by their `x` and `y` coordinates into bins, and applying an aggregation function such as `count` or `sum` (if `z` is provided) to compute the color of the tile representing the bin. This kind of visualization (and the related [2D histogram contour, or density contour](https://plotly.com/python/2d-histogram-contour/)) is often used to manage over-plotting, or situations where showing large data sets as [scatter plots](/python/line-and-scatter/) would result in points overlapping each other and hiding patterns. For data sets of more than a few thousand points, a better approach than the ones listed here would be to [use Plotly with Datashader](/python/datashader/) to precompute the aggregations before displaying the data with Plotly.
```

Comment (emmanuelle) [reply doc/python/2D-Histogram.md:38]: specifed - > specified

**Comment by emmanuelle** [comment]
:dancer: after accepting the two suggestions related to a typo

File: doc/python/2D-Histogram.md (hunk 1 above)
Comment (nicolaskruchten) [reply doc/python/2D-Histogram.md:42]: actually it'll be easier to search and replace if I leave this "standard language" in place as-is ;)

**Pull request merged**